framework/phool/recoConsts.cc

```diff
@@ -163,7 +163,7 @@ void recoConsts::set_defaults()
   set_DoubleFlag("FMAG_HOLE_LENGTH", 27.94);
   set_DoubleFlag("FMAG_HOLE_RADIUS", 1.27);
   set_DoubleFlag("FMAG_LENGTH", 502.92);
-  set_DoubleFlag("Z_UPSTREAM", -500.);
+  set_DoubleFlag("Z_UPSTREAM", -700.);
```

Comment:
How much does this change affect the target-upstream separation (i.e. chi2_target > or < chi2_upstream)?

Reply:
Actually yet, moving the Z_UPSTREAM by 200cm improves the target/upstream separation. See the plot below.

![Snipaste_2024-10-15_02-30-48](https://github.com/user-attachments/assets/c81a39b9-63fb-4ffb-aecf-06fba722634a)
The blue line is $`\chi^{2}_{upstream}-\chi^{2}_{target}`$ when upstream is set to -500 cm, the red line is the same distribution when upstream is moved to -700 cm.

Reply:
While the change improves the efficiency of target/upstream separation, it worsens the purity simultaneously, right?  I remember the beam window is around -500 cm.  The guessed number is fine for now, but we will need to know the locations of upstream objects, and probably run the GMC study.